src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  width: $width;
+  gap: $gap;
+  padding: $padding;
+
+  &__day {
+    width: $size;
+    height: $size;
+    background: #eee;
+    border: 1px solid black;
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index}) {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
```

Comment:
you need to add this styles for all days, so you can use class calendar__day without nth-child